src/common/test-config.c

```diff
@@ -134,11 +133,10 @@ test_get_strvs (void)
 static void
 test_load_dir (void)
 {
-  cockpit_config_dirs[0] = SRCDIR "/src/ws/mock-config";
+  g_setenv("XDG_CONFIG_DIRS", "/var/lib/empty:" SRCDIR "/src/ws/mock-config", 1);
   cockpit_config_file = "cockpit.conf";
 
   g_assert_cmpstr (cockpit_conf_string ("Section2", "value1"), ==, "string");
-  g_assert_cmpstr (cockpit_conf_get_dirs ()[0], ==, SRCDIR "/src/ws/mock-config");
```

Comment:
Why did you drop that assertion? It shoudl still work if you change the [0] to [1], no?

Reply:
ah, misread that as cockpit_config_dirs[]